configuration/debian/tedge-apt-plugin/postrm

```diff
@@ -2,7 +2,4 @@
 
 set -e
 
-if [ ! -e /etc/tedge/sm-plugins/apt && ! -L /etc/tedge/sm-plugins/apt ];
-then
-   rm /etc/tedge/sm-plugins/apt
-fi
+rm -f /etc/tedge/sm-plugins/apt
```

Comment:
I added those checks to make sure that the link exists. If we try to remove it without checking, doesn't the command fail?

Reply:
I put a comment about this in the PR description ;)

> Use rm -f which does not fail if the file does not exist (as this also avoid race conditions between the test and action).

Reply:
Short answer, no `rm -f` does not fail if the file does not exist due to the `-f` option.

https://linux.die.net/man/1/rm

> -f, --force
>     ignore nonexistent files, never prompt

Reply:
I got it, thanks!